app/models/tasks/schedule_hearing_colocated_task.rb

```diff
@@ -36,13 +36,18 @@ def vacols_location
     LegacyAppeal::LOCATION_CODES[:schedule_hearing]
   end
 
-  def create_schedule_hearing_task_on_completion
-    if appeal_type.eql?(Appeal.name) &&
-       saved_change_to_status? &&
-       completed? &&
-       all_tasks_closed_for_appeal? &&
-       assigned_to.is_a?(Organization)
-      ScheduleHearingTask.create!(appeal: appeal, parent: appeal.root_task)
-    end
+  def just_completed?
+    appeal_type.eql?(Appeal.name) &&
+      saved_change_to_status? &&
+      completed? &&
+      all_tasks_closed_for_appeal? &&
+      assigned_to.is_a?(Organization)
+  end
+
+  def send_to_hearings_branch
+    parent = DistributionTask.create!(appeal: appeal, parent: appeal.root_task)
+    ScheduleHearingTask.create!(appeal: appeal, parent: parent)
+    JudgeTask.open.where(appeal: appeal).find_each(&:cancel_task_and_child_subtasks)
+    DistributedCase.find_by(case_id: appeal.uuid).update!(case_id: "#{appeal.uuid}-attempt1")
```

Comment:
Should this be
```suggestion
    DistributedCase.find_by(case_id: appeal.uuid)&.update!(case_id: "#{appeal.uuid}-attempt1")
```
We do not create a distributed case when case movement users advance a case to a judge.